### ISSUE TITLE
Nan loss during training

### ISSUE THREAD

**Issue by Waynechen92** [ENSTA-U2IS/LDU#4]
Has anyone ever met Nan lossing during training for bts_main?
Any possible solution for that?

**Comment by xuanlongORZ** [comment]
Hi, thank you for leaving the comments.
Actually, it appears in the original bts training as well, you can check [here](https://github.com/cleinc/bts/issues/36) to find some suggestions given by the authors of bts.

**Comment by Waynechen92** [comment]
Thanks for your reply xuanlong, I'm also wondering whether it is correct to have negative loss in training for bts_main

**Comment by xuanlongORZ** [comment]
Several common loss functions are indeed non-negative such as MSELoss or L1Loss. But it is fine to have a negative loss value during training, not only in LDU case. You can check [here](https://discuss.pytorch.org/t/negative-value-in-my-loss-function/101776/2) for example.
